handler/src/main/java/com/networknt/handler/config/PathChain.java

```diff
@@ -32,11 +39,37 @@ public String getMethod() {
         return method;
     }
 
-    public void setMethod(String method) throws Exception {
-        if (NetUtils.METHODS.contains(method.toUpperCase())) {
-            this.method = method;
+    public void setMethod(String method) { this.method = method; }
+
+    @Override
+    public String toString() {
+        if(path != null) {
+            return path + "@" + method + " → " + exec;
         } else {
-            throw new Exception("Invalid HTTP method provided: " + method);
+            return source + "() → " + exec;
+        }
+    }
+    /**
+     * Validate the settings and raise Exception on error.
+     * The origin is used to help locate problems.
+     */
+    public void validate(String origin) {
```

Comment:
doesn't seem like there's a check against source and path both being null?

Reply:
That would be addressed here:
```java
        if(source == null) {
            if(path == null) {
                problems.add("You must specify either path or source");
```
I have added a unit test for this as well.

Reply:
Not sure how i missed that before 🙈 

Thanks!